CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+## dev
+
+* `aframe-core` merged with `aframe`. `window.AFRAME` exposes what `aframe-core` was
+exposing previously (e.g., `AFRAME.registerComponent` vs.
+`AFRAME.aframeCore.registerComponent`).
+* Improved Android support.
+* Entities, including the scene, wait for their children to load before emitting the `loaded`
+event.
+* Scene `canvas` elements are now appended to the scene by default rather than
+to the document body. The scene can specify the canvas to render to.
+* Component property types. Property types define how a component property
```

Comment:
i use vim to wrap against a max number of columns. not causing any formatting issues and makes it a bit easier to read in the editor